src/style.css

```diff
@@ -0,0 +1,145 @@
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+
+  --button-color: #00acdc;
+  --text-main-color: #060b35;
+  --text-secondary-color: #616070;
+  --text-white-color: #fff;
+}
+
+.card {
+  margin: 0;
+  box-sizing: border-box;
```

Comment:
Move to * selector and remove it from all other places.

Reply:
didn't fixed
https://github.com/mate-academy/layout_product-cards/pull/3030/files#r1194784920